src/applications/mhv/medications/tests/e2e/pages/MedicationsListPage.js

```diff
@@ -429,5 +429,39 @@ class MedicationsListPage {
         });
       });
   };
+
+  verifyCmopNdcNumberIsNull = () => {
+    // cy.intercept(
+    //   'GET',
+    //   `/my_health/v1/prescriptions/${
+    //     prescriptionDetails.data.attributes.prescriptionId
+    //   }`,
+    //   prescriptionDetails,
+    // ).as('prescriptionDetails');
+    cy.get('@medicationsList')
+      .its('response')
+      .then(res => {
+        expect(res.body.data[1].attributes).to.include({
+          cmopNdcNumber: null,
+        });
+      });
+  };
+
+  verifyPrescriptionSourceForNonVAMedicationOnDetailsPage = () => {
+    // cy.intercept(
+    //   'GET',
+    //   `/my_health/v1/prescriptions/${
+    //     prescriptionDetails.data.attributes.prescriptionId
+    //   }`,
+    //   prescriptionDetails,
+    // ).as('prescriptionDetails');
```

Comment:
Will this block actually be needed later or can it be removed?

Reply:
yes, I can remove the ones you have mentioned here. Thanks!